docs/source/01_tutorials/02_dataframe_model.md

```diff
@@ -0,0 +1,40 @@
+# DataFrameModel
+
+Alternatively, `pandera` support a class-based `DataFrameModel` schema.
+
+An example of schema look like:
+```python
+import pandera as pa
+from pandera.typing import Series
+
+
+class ExampleIrisDataSchema(pa.DataFrameModel):
+    sepal_length: Series[float] = pa.Field(gt=2000)
+```
+
+You should put this in `src/<kedro_pandera_tutorial/schema/example_iris_data.py` and create a `src/kedro_pandera_tutorial/schema/example_iris_data/__init__.py` file.
+
+the `__init__.py` need to import the class.
```

Comment:
I actually mean `src/kedro_pandera_tutorial/schemas/__init__.py`, but it seems like without it, it still works which got me confused.

Without the `__init__.py` it get created as a namespace package. See https://peps.python.org/pep-0420/

Though I think it's still a good practice to make everything import from `schemas/__init__.py`?

Reply:
> Yes. Testing is a pain in the a** because we need to mock an entire kedro repo with a context if we want to have things realistic.

Agree. Mocking files are annoying and we have a lot of tests like this. Internally some team create a `TestProject` class which provide API to mock these stuff for testing.

Reply:
> I actually mean src/kedro_pandera_tutorial/schemas/__init__.py, but it seems like without it, it still works which got me confused.
>
> Without the __init__.py it get created as a namespace package. See https://peps.python.org/pep-0420/
> still 
> Though I think it's still a good practice to make everything import from schemas/__init__.py?

I meant that it should work with any path to a module, not necessarily this exact folder structure. My comment was not about the ``__init__.py``. 

Regarding the ``__init__.py`` being not necessary, I think this is still a best practice. I remember having troubles when submodules did not have the ``__init__.py`` in the past (I remember reading about these namespace package back then, but I don't remember the details) and I don't know well the subtleties of module import in python. Let's still suggest have it.